### PR TITLE
spider web buff

### PR DESCRIPTION
**Pull request by kxvvv**
## About the PR

Now spiders move faster on the web.

**Media**

https://github.com/space-wizards/space-station-14/assets/136233650/b7532b80-e15e-46a3-9665-05e5397be14f

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: Spiders have become more agile!

### CODE REVIEW

**Comment by kxvvv** [comment]
ops

File: Content.Shared/Movement/Systems/FastContactsSystem.cs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    private void OnEntityEnter(EntityUid uid, FastContactsComponent component, ref StartCollideEvent args)
+    {
+        var otherUid = args.OtherEntity;
+        if (!HasComp<MovementSpeedModifierComponent>(otherUid))
+            return;
+
+        EnsureComp<FasterOnContactComponent>(otherUid);
+        _toUpdate.Add(otherUid);
+    }
+}
```
Comment (metalgearsloth) [inline Content.Shared/Movement/Systems/FastContactsSystem.cs:126]: Please don't copypaste systems like this, just use the existing ones in this case.

File: Resources/Prototypes/Entities/Mobs/NPCs/broodmother.yml [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  allowRepeatedMorphs: true
+  delay: 3
+
+
+#action egg
+- type: instantAction
+  id: SpiderLayEgg
+  name: Lay a spider egg
+  description: While your side is falling, you are destroying the throne.
+  icon: { sprite: Mobs/Animals/spideregg.rsi, state: egg }
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/NPCs/broodmother.yml:217]: cl doesn't list any new mobs or events.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
If you're making balance changes please say why.

**Pull request closed without merging**